packages/hash/subgraph/jest.config.cjs

```diff
@@ -0,0 +1,23 @@
+/** @type {import('jest').Config} */
+module.exports = {
+  collectCoverage: process.env.TEST_COVERAGE === "true",
+  collectCoverageFrom: [
+    "**/*.{c,m,}{j,t}s{x,}",
+    "!**/node_modules/**",
+    "!**/dist/**",
+  ],
+  coverageReporters: ["lcov", "text"],
+  preset: "ts-jest",
+  testEnvironment: "node",
+  // recreating DB takes longer than the default 5 seconds.
+  // The chosen default give a lot of room to the integration test.
+  testTimeout: 60000,
+  moduleNameMapper: {
+    "@hashintel/hash-backend-utils(.*)": "<rootDir>/../backend-utils/src$1",
+    "@hashintel/hash-shared(.*)": "<rootDir>/../shared/src$1",
+    "@hashintel/hash-graph-client": "<rootDir>/../../graph/clients/typescript",
+  },
```

Comment:
Apologies, I was only referring to `@hashintel/hash-backend-utils` and `@hashintel/hash-shared`, not `@hashintel/hash-graph-client`.

Perhaps I've not grasped the scope of this package very well, in my mind there wouldn't have been a circumstance for it to need `getRequiredEnv` or `Logger` for instance, and therefore not to be a need for this dependency. Since you removed the dependencies in response to [this thread](https://github.com/hashintel/hash/pull/1383#discussion_r1025088316), I still think they should be removed here as importing from these won't be possible anyway:
```suggestion
  moduleNameMapper: {
    "@hashintel/hash-graph-client": "<rootDir>/../../graph/clients/typescript",
  },
```